civictechprojects/templates/new_index.html

```diff
@@ -39,6 +39,7 @@
           emailVerified: {{emailVerified|yesno:"true,false"}},
           firstName: '{{firstName}}',
           lastName: '{{lastName}}',
+          isStaff: '{{isStaff}}' === "True" ? true : false
```

Comment:
emailVerified is the same sort of value, so let's treat the two consistently, either using emailVerified's 'yesno' construction, or changing it to match isStaff's ternary construction.